content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/mto.rst

```diff
@@ -2,121 +2,127 @@
+   confirmation of an |SO| or |MO|. This is the case, even if there is enough stock of the product
+   on-hand to fulfill the |SO|, without buying or manufacturing additional units of it.
 
-While the |MTO| route can be used in unison with a variety of other routes, the *Buy* route is used
-as the example for this workflow. Begin by navigating to the :menuselection:`Sales` app, then click
-:guilabel:`Create`, which opens a blank quotation form.
+While the |MTO| route can be used in unison with the *Buy* or *Manufacture* routes, the *Buy* route
+is used as the example for this workflow. Begin by navigating to the :menuselection:`Sales` app,
+then click :guilabel:`New`, which opens a blank quotation form.
 
-On the blank quotation form, add a :guilabel:`Customer`, then click :guilabel:`Add a product` under
-the :guilabel:`Order Lines` tab, and enter a product that has been configured to use the *MTO* and
-*Buy* routes. Click :guilabel:`Confirm` and the quotation is turned into a sales order.
+On the blank quotation form, add a :guilabel:`Customer`. Then, click :guilabel:`Add a product` under
+the :guilabel:`Order Lines` tab, and enter a product configured to use the *MTO* and *Buy* routes.
+Click :guilabel:`Confirm`, and the quotation is turned into an |SO|.
 
-A :guilabel:`Purchase` smart button now appears in the top-right corner of the sales order. Clicking
-it opens the |RFQ| associated with the sales order.
+A :guilabel:`Purchase` smart button now appears at the top of the page. Clicking it opens the |RFQ|
+associated with the |SO|.
 
-Click :guilabel:`Confirm Order` to confirm the |RFQ|, and turn it into a purchase order. A green
-:guilabel:`Receive Products` button now appears at the top of the purchase order. Once the products
-are received, click :guilabel:`Receive Products` to register them into inventory.
+Click :guilabel:`Confirm Order` to confirm the |RFQ|, and turn it into a |PO|. A purple
+:guilabel:`Receive Products` button now appears above the |PO|. Once the products are received,
+click :guilabel:`Receive Products` to open the receipt order, and click :guilabel:`Validate` to
+enter the products into inventory.
 
-Return to the sales order by clicking the :guilabel:`SO` breadcrumb, or by navigating to
-:menuselection:`Sales --> Orders --> Orders`, and selecting the sales order.
+Return to the |SO| by clicking the :guilabel:`SO` breadcrumb, or by navigating to
+:menuselection:`Sales app --> Orders --> Orders`, and selecting the|SO|.
 
-Finally, click the :guilabel:`Delivery` smart button in the top-right of the order to be taken to
-the delivery order. Once the products have been shipped to the customer, click :guilabel:`Validate`
-to confirm the delivery.
+Finally, click the :guilabel:`Delivery` smart button at the top of the order to open the delivery
+order. Once the products have been shipped to the customer, click :guilabel:`Validate` to confirm
+the delivery.
```

Comment:
Might be worth adding subcontracting section here as a **seealso**, since it adjacently uses the MTO route.

Reply:
I will add an admonition block with links to any docs that use the MTO route in their workflows.